crates/rpc/rpc-eth-types/src/fee_history.rs

```diff
@@ -398,13 +408,19 @@ impl FeeHistoryEntry {
     ///
     /// See also [`Self::next_block_excess_blob_gas`]
     pub fn next_block_blob_fee(&self) -> Option<u128> {
-        self.next_block_excess_blob_gas().map(alloy_eips::eip4844::calc_blob_gasprice)
+        zip(self.next_block_excess_blob_gas(), self.blob_params)
+            .map(|(excess_blob_gas, params)| params.calc_blob_fee(excess_blob_gas))
+            .next()
     }
 
     /// Calculate excess blob gas for the next block according to the EIP-4844 spec.
     ///
     /// Returns a `None` if no excess blob gas is set, no EIP-4844 support
     pub fn next_block_excess_blob_gas(&self) -> Option<u64> {
-        Some(alloy_eips::eip4844::calc_excess_blob_gas(self.excess_blob_gas?, self.blob_gas_used?))
+        multizip((self.excess_blob_gas, self.blob_gas_used, &self.blob_params))
```

Comment:
why is multizip here required?

unclear what this does, can we not use the ? operator